lib/clients/rtm/client.js

```diff
@@ -403,13 +415,15 @@ RTMClient.prototype._handleWsMessageViaEventHandler = function _handleWsMessageV
   messageType, message) {
   var replyTo;
 
+  if (!isUndefined(this.dataStore)) {
+    this.dataStore.handleRtmMessage(this.activeUserId, this.activeTeamId, messageType, message);
+  }
+
```

Comment:
Yep, that'd be it. Thanks for catching this! I'm not sure where the message with message type of `message` I tested against came from...